tree_test.go

```diff
@@ -21,8 +21,7 @@ var testFiles = []struct {
 	{"bar/bla/blubb", []byte("This is just a test!\n")},
 }
 
-// prepareDir creates a temporary directory and returns it.
```

Comment:
again, renaming the function is more concise than having a comment on a function with an ambiguous name, imho.